chromium_src/chrome/browser/ui/webui/chrome_untrusted_web_ui_configs.cc

```diff
@@ -17,7 +17,7 @@
 
 #if BUILDFLAG(ENABLE_BRAVE_VPN) && !BUILDFLAG(IS_ANDROID)
 #include "brave/browser/ui/webui/brave_vpn/vpn_panel_ui.h"
-#include "brave/components/brave_vpn/brave_vpn_utils.h"
+#include "brave/components/brave_vpn/common/brave_vpn_utils.h"
```

Comment:
brave_vpn_utils should not be inside `!IS_ANDROID`

Reply:
done